src/remediations/fifi.js

```diff
@@ -65,28 +86,39 @@ exports.getSatelliteId = function (facts) {
     return null;
 };
 
-function defineRHCEnabledExecutor (satellites) {
+function defineRHCEnabledExecutor (satellites, smart_management) {
     const satlessExecutor = _.find(satellites, satellite => satellite.id === null);
     if (satlessExecutor) {
-        const partitionedSystems = _.partition(satlessExecutor.systems, system => _.isUndefined(system.rhc_client));
+        const partitionedSystems = sortRHCSystems(satlessExecutor, smart_management);
+
+        if (!_.isEmpty(partitionedSystems[0])) {
+            satellites.push({id: null, systems: partitionedSystems[0], type: 'RHC'});
```

Comment:
Is there a reason not to set `rhcStatus: connected` here and instead have a special clause for that in `fetchRHCStatus`?